src/main/java/org/wise/vle/web/wise5/StudentDataController.java

```diff
@@ -157,6 +152,67 @@ public void getWISE5StudentData(HttpServletResponse response,
     }
   }
 
+  private boolean isAllowedToGetStudentData(User user, Run run, Integer workgroupId)
+      throws ObjectNotFoundException {
+    return isMemberOfWorkgroupId(user, run, workgroupId);
+  }
+
+  private boolean isAllowedToGetEvents(User user, Run run, Integer workgroupId)
+      throws ObjectNotFoundException {
+    return isMemberOfWorkgroupId(user, run, workgroupId);
+  }
```

Comment:
Since these functions both just return the result of `isMemberOfWorkgroupId()`, do we need them or can you just call `isMemberOfWorkgroupId` in the places where they're invoked?